v2/cmd/asoctl/pkg/importresources/resource_importer.go

```diff
@@ -81,116 +83,149 @@ func (ri *ResourceImporter) Import(
 	ctx context.Context,
 	done chan struct{},
 ) (*Result, error) {
-	workersRequired := ri.desiredWorkers()
-
-	candidates := make(chan ImportableResource)  // candidates that need to be deduped
-	pending := make(chan ImportableResource)     // importers that are pending import
-	successes := make(chan ImportResourceResult) // importers that have been executed successfully
-	failures := make(chan ImportError)           // errors from importers that failed
-	completions := make(chan struct{})           // channel to signal completion
+	resources := make(chan ImportableResource) // resources to import
```

Comment:
Thoughts on making this a buffered channel?

Reply:
I don't think it needs to be. 

The deduplicator created in `startDeduplicator` immediately consumes anything put on this channel and does all the buffering we need (and handles the size dynamically too).